bindings/rust/s2n-tls-hyper/tests/web_client.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use bytes::Bytes;
+use http::{status, Uri};
+use http_body_util::{BodyExt, Empty};
+use hyper_util::{client::legacy::Client, rt::TokioExecutor};
+use s2n_tls::config::Config;
+use s2n_tls_hyper::connector::HttpsConnector;
+use std::{error::Error, str::FromStr};
+
+#[tokio::test]
+async fn test_get_request() -> Result<(), Box<dyn Error>> {
+    let connector = HttpsConnector::builder(Config::default()).build();
+    let client: Client<_, Empty<Bytes>> = Client::builder(TokioExecutor::new()).build(connector);
+
+    let uri = Uri::from_str("https://www.amazon.com")?;
```

Comment:
Adding tests that send/receive HTTP requests locally seemed involved, so I planned to add the ability to do this in a separate PR. Since sending an HTTP request externally is currently the only way to test the HttpsConnector, would it make sense to keep this test until I add a way to do this locally?

Alternatively, I could add the local testing module to this PR but I worry it might get too big.

Either way, I removed the external HTTP call from the doc test to avoid flakiness.

Reply:
It should be fine as long as it's temporary. Is there an issue tracking removing it?

Reply:
I created one: https://github.com/aws/s2n-tls/issues/4631